### ISSUE TITLE
(Netplay) Warnings must be disabled if there is a server password

### ISSUE THREAD

**Issue by furiadeoso** [libretro/RetroArch#6965]
## Description

When someone is hosting a public netplay and set up a server password nobody can enter, but each failed attempt to enter is prompted in the host screen as a warning, showing the nick of the user.

This is an undesired behavior since currently the number of lobby users is increasing, and many ignore the status of the password protected server. 

So many times, many annoying messages can be received on the screen during a game.

### Expected behavior
Users of a pasword protected netplay must not be disturbed by warnings after failed passwords attempts. Nevertheless, if a client enters a  right password, warnings must be showed as usual.

**Comment by RafaelSa94** [comment]
Is anyone working on this? If not, I can do it

**Comment by furiadeoso** [comment]
Thanks for your interest, @RafaelSa94 :)

**Comment by natinusala** [comment]
Why not remove this warning that has no reason to be here in the first place?

**Comment by furiadeoso** [comment]
> Why not remove this warning that has no reason to be here in the first place?

Yeah. Removing this warning when password protection is ON, would be the main goal. 
This is what happens when someone try to enter:
![Trial to enter](https://user-images.githubusercontent.com/34688502/58326120-24a58980-7e2c-11e9-8417-c7907dc39eb2.png)

**Comment by LibretroAdmin** [comment]
@Cthulhu-throwaway Can this be closed?
